apps/src/code-studio/initApp/project.js

```diff
@@ -1914,7 +1900,14 @@ var projects = (module.exports = {
         .then(({data, jqXHR}) => {
           currentSourceVersionId =
             jqXHR && jqXHR.getResponseHeader('S3-Version-Id');
-          unpackSources(data);
+          unpackSources(data, true);
+        })
+        .catch(err => {
+          this.logError_(
```

Comment:
I don't expect this to get hit often/ever, but I ran into an issue in testing where an error in `unpackSources` was getting swallowed.